src/Resources/translations/SonataMediaBundle.ca.xliff

```diff
@@ -0,0 +1,568 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" version="1.2">
+    <file source-language="en" datatype="plaintext" original="file.ext">
```

Comment:
`file.ext`?

Reply:
what is this about

Reply:
I copied it from the `en` version.

I fixed the catalan one and also all the other files :)